README.md

```diff
@@ -375,5 +375,15 @@ bird.singMatingCall();
 
 ```
 
+### @instrument
+
+Uses `console.time` and `console.timeEnd` to provide function timings with a unique label whose default prefix is `ClassName.method`.
+
+Will polyfill `console.time` if the current environment does not support it. You can also supply a custom `console` object with the following methods:
```

Comment:
Doesn't describe the first argument, prefix. What about instead having a signature of:

```js
class Foo {
  @instrument({ prefix: 'something', console: customConsole })
  bar() {}
}
```